doc/release/release_0.20.rst

```diff
@@ -0,0 +1,507 @@
+  The new properties ``num_pixels`` and ``coords_scaled`` are available as well.
+  See the respective docstrings for more details
+  (`#6296 <https://github.com/scikit-image/scikit-image/pull/6296>`_).
+- Add support for the Modified Hausdorff Distance (MHD) metric in ``skimage.metrics.hausdorff_distance`` via the new parameter ``method``.
+  The MHD can be more robust against outliers than the directed Hausdorff Distance (HD)
+  (`#5581 <https://github.com/scikit-image/scikit-image/pull/5581>`_).
+- Added two datasets ``skimage.data.protein_transport`` and ``skimage.data.nickel_solidification``
+  (`#6087 <https://github.com/scikit-image/scikit-image/pull/6087>`_).
+- Incorporate RANSAC improvements from scikit-learn into ``skimage.measure.ransac`` which should lower iteration times
+  (`#6046 <https://github.com/scikit-image/scikit-image/pull/6046>`_).
+- Add the new parameter ``use_gaussian_derivatives`` to ``skimage.feature.hessian_matrix`` which allows the computation of the Hessian matrix by convolving with Gaussian derivatives
+  (`#6149 <https://github.com/scikit-image/scikit-image/pull/6149>`_).
+- Allow footprints with non-adjacent pixels as neighbors in ``skimage.morphology.flood_fill``
+  (`#6236 <https://github.com/scikit-image/scikit-image/pull/6236>`_).
+- Add new parameters ``squared_butterworth`` and ``npad`` to ``skimage.filters.butterworth``, which add support for squaring the filter and edge padding
+  (`#6251 <https://github.com/scikit-image/scikit-image/pull/6251>`_).
+- Add support for n-dimensional images in ``skimage.filters.farid`` (Farid & Simoncelli filter)
+  (`#6257 <https://github.com/scikit-image/scikit-image/pull/6257>`_).
+- Support the construction of ``skimage.io.ImageCollection`` from a ``load_pattern`` with an arbitrary sequence as long as a matching ``load_func`` is provided
+  (`#6276 <https://github.com/scikit-image/scikit-image/pull/6276>`_).
+- Warn for non-integer image inputs to ``skimage.feature.local_binary_pattern``.
+  Applying the function to floating-point images may give unexpected results when small numerical differences between adjacent pixels are present
+  (`#6272 <https://github.com/scikit-image/scikit-image/pull/6272>`_).
+- Use the minimal required unsigned integer size in ``skimage.filters.rank_order`` which allows to operate the function with higher precision or on larger arrays.
+  Previously, the returned ``labels`` and and ``original_values`` were always of type uint32.
+  (`#6342 <https://github.com/scikit-image/scikit-image/pull/6342>`_).
+- Use the minimal required unsigned integer internally in ``skimage.morphology.reconstruction`` which allows to operate the function with higher precision or on larger arrays.
+  Previously, int32 was used.
+  (`#6342 <https://github.com/scikit-image/scikit-image/pull/6342>`_).
```

Comment:
@grlee77, could you please have look if I correctly summarized https://github.com/scikit-image/scikit-image/pull/6342?

> single precision seed now returns single precision output (this function was missed during previous float32 work)

I'm confused about this bit, because I don't see any changes in the PRs diff that should impact seed?

Reply:
This was just referring to a place where `morphology.reconstruct` had unconditional promotion to float64 internally. See at the left in the diff here on lines 173-175 where `seed` and `mask` used to get copied into a float64 array. This was changed to use `_supported_float_type` so it is possible to get single precision output from this function.

https://github.com/scikit-image/scikit-image/pull/6342/files#diff-a0783c211f0b0a66e4cc41bf1838cdafe46ab957a9dfdb685784aecd64135243L173

Reply:
Thanks for the clarification. So actually `mask` determines the dtype of the output and not `seed`?

Reply:
Good question... I would assume we would want "float_dtype = max { _supported_float_type(seed.dtype), _supported_float_type(mask.dtype) }" :thinking: